src/products/degen-mode.md

```
@@ -0,0 +1,42 @@
+## Options Contract Multiplier:100::
+
+It is important to remember that [Options Contract Multiplers](https://www.google.com/search?q=What+is+an+options+contract+multiplier%3F&sca_esv=33dcfe7d0c1ffd3e&rlz=1C1UEAD_enUS1166US1166&ei=5MvFaPePI9STwbkP-tCbuAQ&ved=0ahUKEwj3kaKTwtaPAxXUSTABHXroBkcQ4dUDCBE&uact=5&oq=What+is+an+options+contract+multiplier%3F&gs_lp=Egxnd3Mtd2l6LXNlcnAiJ1doYXQgaXMgYW4gb3B0aW9ucyBjb250cmFjdCBtdWx0aXBsaWVyPzIFECEYoAEyBRAhGKABMgUQIRigATIFECEYoAEyBRAhGKABMgUQIRirAkjHP1AAWOc9cAB4AZABAJgBe6AB2RqqAQQzMS44uAEDyAEA-AEBmAInoAK2G8ICCxAAGIAEGJECGIoFwgIKEAAYgAQYQxiKBcICDhAuGIAEGLEDGNEDGMcBwgILEAAYgAQYsQMYgwHCAggQABiABBixA8ICERAuGIAEGLEDGNEDGIMBGMcBwgIEEAAYA8ICDhAAGIAEGLEDGIMBGIoFwgIOEC4YgAQYsQMYgwEYigXCAgUQABiABMICBhAAGBYYHsICCBAAGBYYChgemAMAkgcEMzEuOKAHleoBsgcEMzEuOLgHthvCBwY2LjI5LjTIB0o&sclient=gws-wiz-serp) exist - traditionally meaning each value must be multiplied 100x. 
+
+Thanks to to DeFi, Buyers & Sellers will be able to determine whatever size they want for their position through [fractionalization.](/option-standard-sdk/opm#fractionalized-options-for-buyers-and-sellers)  
+> For example: 
+>> selecting a quantity of 1 = 100x of the underlying asset  
+>> Selecting a quantity of 0.5 = 50x of the underlying asset
+>>
+>> Selecting a quantity of 0.25 = 25x of the underlying asset
+
+## Options Tips for Degens :thinking::thought_balloon::
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '::'

(MD026, no-trailing-punctuation)